Makefile

```diff
@@ -123,6 +131,17 @@ $(OC) oc: $(TOOLS_DIR)
 		$(OC) version | grep -q $${version##v} ;\
 	}
 
+.PHONY: jsonnet-tools
+$(JSONNET) jsonnet-tools: $(TOOLS_DIR)
+	@{ \
+		set -ex ;\
+		[[ -f $(JSONNET) && -f $(JSONNETFMT) && -f $(JSONNET_BUNDLER) && -f $(GOJSONTOYAML) ]] && exit 0 ;\
+		GOBIN=$(TOOLS_DIR) go install github.com/google/go-jsonnet/cmd/jsonnet@latest ;\
+		GOBIN=$(TOOLS_DIR) go install github.com/google/go-jsonnet/cmd/jsonnetfmt@latest ;\
+		GOBIN=$(TOOLS_DIR) go install -a github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb@latest ;\
+		GOBIN=$(TOOLS_DIR) go install github.com/brancz/gojsontoyaml@latest ;\
+	}
+
 # Install all required tools
 .PHONY: tools
 tools: $(CONTROLLER_GEN) \
```

Comment:
The jsonnet-tools need to be added here. 
And the `.github/tools` need to be updated. 
We also need .github/tools-cache/action.yaml to be updated so that the cache is correctly maintained.

Reply:
I had added that initially and removed it. I thought to add once I add the ci changes also like to do lining tests and generate tests in GitHub actions. 

Do we add multiple types of commit in pr in repo since we follow conventional commits I thought to add that in another pr. But I can add in same pr if that's ok

Reply:
It's fine to add multiple commits, and they can all have different types if you wish